app/assets/javascripts/libs/globals.js

```diff
@@ -0,0 +1,9 @@
+window.$ = window.jQuery = require('jquery');
+window.I18n = require('i18n-js');
+window._ = require('underscore');
```

Comment:
Yes :/, we need `$` because of attachinary, that isn't on NPM (maybe we could send a PR to their repository) and `I18n` because of the I18n global generated by the gem with the I18n data. `_` is being used by the EJS's, the EJS loader doesn't require recursively.